packages/quantic/cypress/integration/facets/facet/facet-expectations.ts

```diff
@@ -0,0 +1,34 @@
+import {InterceptAliases} from '../../../page-objects/search';
+import {BaseFacetSelector} from '../facet-common-selectors';
+import {FacetSelectors} from './facet-selectors';
+
+export function expectFacetValueContains(
+  selector: BaseFacetSelector,
```

Comment:
Is this necessary or can we use FacetSelectors instead?

Reply:
That's a good question.  So, this is mostly a copy-paste from atomic. My understanding is that we can't put `FacetSelectors` right there because the expectation is shared by the various facet types, and obviously they expose different features. Going this route would force the type to grow and include all features of all facet types. That would make the typing basically useless because the type would not reflect the facet capabilities anymore.

Reply:
Ah yes i see. Thanks